docs/quickstart.md

```diff
@@ -13,32 +13,463 @@ You can find the latest Sedna release [here](https://github.com/kubeedge/sedna/r
+tar -zxvf video.tar.gz
+ 
+```
+
+#### 2.3.3 Scripts prepare
+
+In incremental_learning jobs, the following scripts will be indispensable:
+
+- train.py: script for model fine-tuning/training.
+- eval.py: script for model evaluate.
+- inference.py: script for data inference.
+
+You can also find demos [here](/examples/incremental_learning/helmet_detection).
+
+Some interfaces should be learn in job pipeline:
+
+- `BaseConfig` provides the capability of obtaining the config from env
+
+```python
+
+from sedna.common.config import BaseConfig
+
+train_dataset_url = BaseConfig.train_dataset_url
+model_url = BaseConfig.model_url
+
+```
+
+- `Context` provides the capability of obtaining the context from CRD
+
+```python
+from sedna.common.config import Context
+
+obj_threshold = Context.get_parameters("obj_threshold")
+nms_threshold = Context.get_parameters("nms_threshold")
+input_shape = Context.get_parameters("input_shape")
+epochs = Context.get_parameters('epochs')
+batch_size = Context.get_parameters('batch_size')
+
+```
+
+- `datasources` base class, as that core feature of sedna require identifying the features and labels from data input, we specify that the first parameter for train/evaluate of the ML framework
+
+```python
+from sedna.datasources import BaseDataSource
+
+
+train_data = BaseDataSource(data_type="train")
+train_data.x = []
+train_data.y = []
+for item in mnist_ds.create_dict_iterator():
+     train_data.x.append(item["image"].asnumpy())
+     train_data.y.append(item["label"].asnumpy())
+```
+
+-  `sedna.core` contain all edge-cloud features,  Please note that each feature has its own parameters.
+- **Hard Example Mining Algorithms** in IncrementalLearning named `hard_example_mining`  
+
+```python
+from sedna.core.incremental_learning import IncrementalLearning
+
+hard_example_mining = IncrementalLearning.get_hem_algorithm_from_config(
+    threshold_img=0.9
+)
+
+# initial an incremental instance
+incremental_instance = IncrementalLearning(
+    estimator=Estimator,
+    hard_example_mining=hem_dict
+)
+
+# Call the interface according to the job state
+
+# train.py
+incremental_instance.train(train_data=train_data, epochs=epochs,
+                          batch_size=batch_size,
+                          class_names=class_names,
+                          input_shape=input_shape,
+                          obj_threshold=obj_threshold,
+                          nms_threshold=nms_threshold)
+
+# inference
+results, _, is_hard_example = incremental_instance.inference(
+            data, input_shape=input_shape)
+            
+
+```
+
+
+### 3. Configuration
+
+##### 3.1 Prepare Image
+This example uses the image:  
+```
+kubeedge/sedna-example-incremental-learning-helmet-detection:v0.4.0
+```
+
+This image is generated by the script [build_images.sh](/examples/build_image.sh), used for creating training, eval and inference worker.  
+
+##### 3.2 Create Incremental Job
```

Comment:
Why do you duplicate the example of incremental job here? How about just linking here?

Reply:
This document describes how to quickly develop an application based on sedna. After scripting is complete, the following describes how to run. Diff to example, we removed requirements and results.